src/components/cylc/tree/TreeItem.vue

```diff
@@ -152,7 +152,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
               :key="jobDetail.id"
               class="leaf-entry"
             >
-              <span class="px-4 leaf-entry-title">{{ jobDetail.title }}</span>
+              <span class="px-4 leaf-entry-title" v-html="jobDetail.title"></span>
```

Comment:
I've got a vague memory of seeing a diff that looked like the opposite of this in a recent PR but can't remember which PR. Will try and find out...

Reply:
Ah wait, is it because of the `&Delta;`

Reply:
Yes, it's the `&Delta;` - writing "delta" as a word seems a little primitive for a web page.

Reply:
UTF-8 encoding might be more appropriate?